product_docs/docs/efm/4/upgrading.mdx

```diff
@@ -54,9 +51,6 @@ Upgrade of files is finished. The owner and group for properties and nodes files
 
 If you're [using a Failover Manager configuration without sudo](04_configuring_efm/04_extending_efm_permissions/#running_efm_without_sudo), include the `-source` flag and specify the name of the directory in which the configuration files reside when invoking `upgrade-conf`. If the directory isn't the configuration default directory, the upgraded files are created in the directory from which the `upgrade-conf` command was invoked.
 
-!!! Note
-    If you're using a unit file, manually update the file to reflect the new Failover Manager service name when you perform an upgrade.
-
```

Comment:
From commit msg, an upgrade doesn't change the service name, and it creates a new unit file.